source/includes/auto-encryption-options.rst

```diff
@@ -0,0 +1,147 @@
+The following table describes the structure of an
+``AutoEncryptionOptions`` object:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Name
+     - Description
+
+   * - **keyVaultClient**
+     - | **Type:** `MongoClient <{+api+}/classes/MongoClient.html>`__
+       |
+       | **Description**
+       | A ``MongoClient`` configured to connect to
+       | the MongoDB instance hosting your {+key-vault-long+}, used to
+       | read and write your {+dek-long+}s.
+       |
+       | Specify ``keyVaultClient`` if your {+dek-long+}s are in a
+       | different cluster than your encrypted data.
+       |
+       | To learn more about {+key-vault-long+}s, see :ref:`qe-reference-key-vault`.
+       |
+       | **Default:** ``undefined``
+       |
+       | **Accepted Values:** A configured ``MongoClient``
```

Comment:
Issue: I'm not certain, but does "configured MongoClient" mean configured for encryption?

Suggestion: use the same terminology as earlier "configured for encryption" if that statement is correct.